src/librustdoc/passes/collect_trait_impls.rs

```diff
@@ -204,19 +205,20 @@ impl DocVisitor for ItemCollector {
     }
 }
 
-struct BadImplStripper {
+struct BadImplStripper<'a> {
     prims: FxHashSet<PrimitiveType>,
     items: FxHashSet<ItemId>,
+    crate cache: &'a Cache,
```

Comment:
Why is this crate-public?

```suggestion
    cache: &'a Cache,
```